### DISCUSSION TITLE
collect2: error: ld returned 1 exit status

### DISCUSSION THREAD

**Discussion by entityThiex** [mriscoc/Ender3V2S1#1329] [Q&A]
I've run into an, for me, unknown error. Is this something that is a platformio error or a file depending error? Error in the picture
![image](https://github.com/user-attachments/assets/0cf7696b-05dd-4c33-bc4c-cafde69ba356)

**Comment by mriscoc** [comment]
Possibly an unsupported configuration, be sure tu use the Configurator to generate the configuration files.
